init-tools.sh

```diff
@@ -2,7 +2,7 @@
 
 __scriptpath=$(cd "$(dirname "$0")"; pwd -P)
 __init_tools_log="$__scriptpath/init-tools.log"
-__PACKAGES_DIR="$__scriptpath/packages"
+__PACKAGES_DIR="~/.nuget/packages"
```

Comment:
`$Home` isn't set for all machines...  We shouldn't be using `~` in our scripts.

Reply:
But if we are going to use `$HOME`, we should probably use the actual variable `$HOME`, see https://stackoverflow.com/questions/5930671/why-use-home-over-tilde-in-a-shell-script

Reply:
I plan to use them same behavior as on Windows and ask nuget for the cache directory.